scripts/macpython-install-python.sh

```diff
@@ -351,7 +348,19 @@ function make_workon_venv {
 #echo "Remove and update Python files at ${MACPYTHON_FRAMEWORK}"
 #sudo rm -rf ${MACPYTHON_FRAMEWORK}
 
-for pyversion in $LATEST_3p9 $LATEST_3p10 $LATEST_3p11 $LATEST_3p12 $LATEST_3p13; do
-  install_macpython $pyversion 11
-  install_virtualenv
-done
+if test "$(arch)" == "arm64"; then
+  echo "we are arm"
+  PLAT=arm64
```

Comment:
@thewtex I had moved the setting of PLAT to the top of this file and removed the redundant code.